airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorMetadataDefinitionV0.yaml

```diff
@@ -53,6 +53,16 @@ properties:
         format: uri
       githubIssueLabel:
         type: string
+      maxSecondsBetweenMessages:
+        description: Number of seconds allowed between 2 airbyte protocol messages. The source will timeout if this delay is reach
+        type: integer
+      releaseDate:
+        description: The date when this connector was first released, in yyyy-mm-dd format.
+        type: string
+        format: date
+      protocolVersion:
```

Comment:
I think this field deserve an extra validation on the metadata validation side

Reply:
Thats a good call out! Im thinking a new issue for that. WDYT?